build.sbt

```diff
@@ -52,7 +52,7 @@ libraryDependencies ++= Seq(
   ws exclude("org.apache.httpcomponents", "httpclient") exclude("org.apache.httpcomponents", "httpcore"),
   "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "5.18.0",
   "uk.gov.hmrc" %% "http-caching-client" % "9.5.0-play-28",
-  "uk.gov.hmrc" %% "mongo-caching" % "7.1.0-play-28",
+  "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28" % "0.51.0",
```

Comment:
Same as the FE comment, can we use the latest version, which seems to be 0.62.0?

Reply:
Fixed